bin/pipeline.sh

```diff
@@ -14,6 +15,9 @@ trap 'rm -f $QGRAPH_FILE' EXIT
 FAKES_QGRAPH_FILE=$(mktemp)_fakes.qgraph
 trap 'rm -f $FAKES_QGRAPH_FILE' EXIT
 
+FARO_QGRAPH_FILE=$(mktemp)_faro.qgraph
+trap 'rm -f $FARO_QGRAPH_FILE' EXIT
+
 pipetask qgraph \
```

Comment:
Since you are here, can you add `--long-log` to each of the previous pipetask commands as well?